apps/stress-test/webpack.config.js

```diff
@@ -0,0 +1,130 @@
+const path = require('path');
```

Comment:
@spmonahan please correct me if am wrong, but I don't see the usage of path aliases in this app. So to start it I will need to build whole repo first (WC, v8, v9), is it correct?

Reply:
I don't understand the question. I'm using the `path` module to construct file paths for things like output files.

This application does depend on the v8, v9 and WC implementations of Fluent so those will need to be built in order to start this app.

Reply:
> I don't understand the question. I'm using the path module to construct file paths for things like output files.

I meant this part that we have in Storybook config:
https://github.com/microsoft/fluentui/blob/5a44edc603012847bbe5cc7b29d65bb86c7fb8b2/.storybook/main.js#L55-L61

Setup that we use for `ssr-tests-v9` also uses them as `esbuild` supports them.

> This application does depend on the v8, v9 and WC implementations of Fluent so those will need to be built in order to start this app.

I see, it was actually my question 🐱 Sorry if it was unclear.

Thanks for clarifying, now it's clear that it's required to build all Fluent UI versions first to run the app itself. With path aliases we can avoid it and make the experience more smooth: currently after every change in a component rebuilt is required.

Reply:
> With path aliases we can avoid it and make the experience more smooth: currently after every change in a component rebuilt is required.

That sounds great! I'll look at adding this.

Reply:
I've added this but it looks like the base TS Config only covers v9. Also, I'm not sure how to test that it's working as expected. The app still runs locally but it also worked fine before :)

Definitely appreciate your help here!

Reply:
@Hotell can you please help there?

Reply:
yeah path aliases are used only for v9. this will be enabled also for v8 in the future.

dunno what's the web components story but path aliases will not be implemented for v0.

this means that aliasing should work for v9 as you did setup , but rest needs to be build manually.